vault/changelog.release.2022-02-15.md

```diff
@@ -0,0 +1,94 @@
+---
+id: egxM8X43CqrQLuhYQ6Wu8
+title: '0.82'
+desc: ''
+updated: 1644943065435
+created: 1644902103166
+---
+
+Dendron 0.82 has sprouted  🌱
+
+[[Schema Template Variables|dendron://dendron.dendron-site/dendron.topic.templates.schema-template#template-variables]] can now include `CURRENT_WEEK`!
+
+![[dendron://dendron.dendron-site/dendron.topic.templates.schema-template#template-variables,1:#*]]
```

Comment:
no need for note reference. would actually just remove schema template variables all together - they are already called out in highlights